doc/Reference/Navigation/ViewMap.md

```diff
@@ -1,6 +1,18 @@
 ---
 uid: Reference.Navigation.ViewMap
 ---
+
 # What is a ViewMap
 
-- Define what a viewmap is and the relationship between view, viewmodel and datamap
+When registering routes, we can take advantage of the `ViewMap` object and its variations `DataViewMap` and `ResultDataViewMap`, to associate Views with ViewModels, specify the type of parameters ViewModels may take, and specify the type of return coming from a navigation.
+
+## ViewMap
+
+| Component      | Description          | Example |
+|----------------|----------------------|---------|
+| View           | The type of the View | `new ViewMap(typeof(MainPage))`<br><br>Alternatively can be specified as a generic argument: `new ViewMap<MainPage>()` |
+| ViewSelector   |         ?             | ? |
```

Comment:
Will sync with Nick regarding content for this part.

Reply:
@eriklimakc we'll need to sync to finish this section off